src/theme/theme.ts

```diff
@@ -8,7 +8,7 @@ const size = {
     desktop: '2560px',
 };
 
-const theme = {
+const themes = {
```

Comment:
czemu nazwałaś to themes? Skoro masz jeden theming, to może lepiej nazwać to theme?